omnikinverter/tcp.py

```diff
@@ -272,10 +272,17 @@ def int_to_bool(num: int) -> bool:
         "firmware_slave": None,
     }
 
-    result = {}
+    result: dict[str, Any] = {}
 
     for (name, extractor) in field_extractors.items():
         value = getattr(tcp_data, name)
+
+        # Set temperature to None if it matches 65326, this is returned
+        # when the inverter is "offline".
+        if name == "temperature" and value == 65326:
+            result["temperature"] = None
```

Comment:
The idea was to keep such implementations as a function in the `field_extractors` mapping, but that wasn't possible for `ac_output` which pokes at the whole result map.

Reply:
i'll move it